### ISSUE TITLE
Migrations need to be updated to Rails 5.x

### ISSUE THREAD

**Issue by seanfcarroll** [simukappu/activity_notification#96]
```
StandardError: An error has occurred, this and all later migrations canceled:

Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migration was written for:

  class CreateActivityNotificationTables < ActiveRecord::Migration[4.2]
/Users/sean/src/immersive/immersive/db/migrate/20181128164307_create_activity_notification_tables.rb:2:in `<main>'
/Users/sean/src/immersive/immersive/bin/rails:9:in `<top (required)>'
/Users/sean/src/immersive/immersive/bin/spring:15:in `<top (required)>'
bin/rails:3:in `load'
bin/rails:3:in `<main>'

Caused by:
StandardError: Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migration was written for:

  class CreateActivityNotificationTables < ActiveRecord::Migration[4.2]
```

**Comment by seanfcarroll** [comment]
Fixed by adding the Rails version.

```
class CreateActivityNotificationTables < ActiveRecord::Migration[5.2]
```

**Comment by simukappu** [comment]
Thank you for your reporting! We will fix it. In addition, we have to add migration test with generated migration file from the template.

**Comment by simukappu** [comment]
Fixed and released as v1.7.0.
